docs/src/tutorials/huggingface.rst

```diff
@@ -0,0 +1,183 @@
+*****************************************************************************************
+Hyperparameters optimisation using a HuggingFace Model and the Hydra-Orion-Sweeper plugin
+*****************************************************************************************
+
+In this tutorial, we will show an easy Orion integration of a HuggingFace translation model using
+Hydra, with the `Hydra_Orion_Sweeper <https://github.com/Epistimio/hydra_orion_sweeper>`_ plugin.
+Hydra is essentially a framework for configuring applications. We will use it to define our
+Hyperparameters and some Orion configuration. We will also be using
+`Comet <https://www.comet.com/>`_ for monitoring our experiments.
+
+Imports
```

Comment:
There should be a sections on installation. Adding a requirements.txt file and pointing to it would suffice.

Reply:
added in [4a5965d](https://github.com/Epistimio/orion/pull/1040/commits/4a5965dc093261b1747a1a06d75cd67970f9262a)